backend/src/main/java/com/bakdata/conquery/sql/conversion/forms/HanaStratificationFunctions.java

```diff
@@ -104,6 +116,38 @@ public Table<Record> generateIntSeries(int start, int end) {
 		return DSL.table("SERIES_GENERATE_INTEGER({0}, {1}, {2})", INCREMENT, adjustedStart, end);
 	}
 
+	@Override
+	public Field<Date> indexSelectorField(TemporalSamplerFactory indexSelector, ColumnDateRange validityDate) {
+		return switch (indexSelector) {
+			case EARLIEST -> DSL.min(validityDate.getStart());
+			// end date of validity dates is exclusive, but we need the inclusive one
+			case LATEST -> functionProvider.addDays(DSL.max(validityDate.getEnd()), DSL.val(-1));
+			case RANDOM -> {
+				// TODO seed required?
```

Comment:
Ich glaube wir hatten darüber mal gesprochen, dass es einen Seed dafür bei euch gibt?

Reply:
Genau, der ist als Konstante in ConqueryConstants.RANDOM_SEED festgelegt.

Es war nur wichtig, dass der Seed konstant war zur Reproduzierbarkeit von eventuellen Fehlern.

Reply:
Also das Problem ist, dass nach meinem Kenntnisstand für HANA keine Möglichkeit existiert einen Seed zu setzen. Für Postgres schon.